test/Tests/Readers/LaTeX.hs

```diff
@@ -137,6 +141,42 @@ tests = [ testGroup "tokenization"
           , "Table with vertical alignment argument" =:
             "\\begin{tabular}[t]{r|r}One & Two\\\\ \\end{tabular}" =?>
             simpleTable' [AlignRight,AlignRight] [[plain "One", plain "Two"]]
+          , "Table with multicolumn item" =:
+            "\\begin{tabular}{l c r}\\multicolumn{2}{c}{One} & Two\\\\ \\end{tabular}" =?>
+            table' [AlignLeft, AlignCenter, AlignRight]
+                   [ Row nullAttr [ cell AlignCenter (RowSpan 1) (ColSpan 2) (plain "One")
+                                  , simpleCell (plain "Two")
+                                  ]
+                   ]
+          , "Table with multirow item" =:
+            "\\begin{tabular}{c}\\multirow{2}{c}{One}\\\\Two\\\\\\end{tabular}"=?>
+            table' [AlignCenter]
+                  [ Row nullAttr [ cell AlignCenter (RowSpan 2) (ColSpan 1) (plain "One") ]
+                  , Row nullAttr [ simpleCell (plain "Two") ]
+                  ]
+          , "Table with nested multirow/multicolumn item" =:
+            "\\begin{tabular}{c c c}\\multirow{2}{c}{\\multicolumn{2}{c}{One}}&Two\\\\Three\\\\Four&Five&Six\\\\\\end{tabular}" =?>
```

Comment:
A common pattern in this file is to use `T.unlines […]` as a means to avoid overlong lines.